users: Added user details and teams

StealthyCoder opened the pull request:
Extended the users command and added new teams command.

Signed-off-by: Eric Bode <eric.bode@foundries.io>

doanac reviewed (commented):
Looks good. I'll merge this once we get the backend changes in place.

you can't comment on commit messages, but
```
Extended the users command and added new teams command.
```
If you find yourself saying "and" like that in a commit message - you usually have 2 commits. Not a big deal for this, just something for the future.

StealthyCoder commented:
> Looks good. I'll merge this once we get the backend changes in place.
> 
> you can't comment on commit messages, but
> 
> ```
> Extended the users command and added new teams command.
> ```
> 
> If you find yourself saying "and" like that in a commit message - you usually have 2 commits. Not a big deal for this, just something for the future.

You are right. I just changed the commit history.
